components/page-home.js

```diff
@@ -20,7 +20,7 @@ const HomePage = () => {
                 <SearchForm />
                 <div className="wide-layout two-part-overlap">
                     <div className="content-block-wrap">
-                        <img src={imgTents} />
+                        <img src={imgTents} alt="tents"/>
```

Comment:
This was another intentional issue in a component... that way it will be flagged by cypress-axe.

Reply:
fixing